benchmarks/benchmarks/bench_core.py

```diff
@@ -98,7 +98,7 @@ def time_large2(self):
 
 class CorrConv(Benchmark):
     params = [[50, 1000, 1e5],
-              [10, 100, 1000, 1e4],
+              [10, 100, 1000, int(1e4)],
```

Comment:
Pretty sure you need this on the line above too.

Reply:
Thanks, saved me waiting for travis to run